modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -389,12 +366,26 @@ const setup = (editor: Editor): RenderInfo => {
     return parsedHeight;
   };
 
+  const setupShortcutsAndCommands = (outerContainer: AlloyComponent): void => {
+    editor.addShortcut('alt+F9', 'focus menubar', () => {
+      OuterContainer.focusMenubar(outerContainer);
+    });
+
+    editor.addShortcut('alt+F10', 'focus toolbar', () => {
+      OuterContainer.focusToolbar(outerContainer);
+    });
+
+    editor.addCommand('ToggleToolbarDrawer', () => {
```

Comment:
Should we make another ticket to move the commands to `api/Commands.ts` there is also commands in the ColorSwatch.ts file would be nice to have one spot where they are defined like Options since they are kind of public api.

Reply:
Will log a Jira for the first part and as for the rest, this was a timebox task so I'd largely skipped looking at tests because of that. However, since we have an extension now I'll go ahead and do them.

Reply:
Have added some simple regression tests that should cover this and have logged the commands thing as TINY-8437. Let me know if you want to see anything more added.